does not install on Windows CI

ughh

```
2020-02-20T21:14:50.8000322Z You can now open Cypress by running: node_modules\.bin\cypress open
2020-02-20T21:14:50.8000606Z 
2020-02-20T21:14:50.8000970Z https://on.cypress.io/installing-cypress
2020-02-20T21:14:50.8004545Z 
2020-02-20T21:14:51.0643263Z [?25h
2020-02-20T21:14:51.0643984Z > fsevents@1.2.11 install d:\a\testing-workshop-cypress\testing-workshop-cypress\node_modules\fsevents
2020-02-20T21:14:51.0644374Z > node-gyp rebuild
2020-02-20T21:14:51.0644609Z 
2020-02-20T21:14:51.0767861Z 
2020-02-20T21:14:51.0772351Z d:\a\testing-workshop-cypress\testing-workshop-cypress\node_modules\fsevents>if not defined npm_config_node_gyp (node "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild ) 
2020-02-20T21:14:57.7567169Z Traceback (most recent call last):
2020-02-20T21:14:57.7567909Z   File "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\gyp\gyp_main.py", line 50, in <module>
2020-02-20T21:14:57.7568240Z     sys.exit(gyp.script_main())
2020-02-20T21:14:57.7568648Z   File "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 554, in script_main
2020-02-20T21:14:57.7568927Z     return main(sys.argv[1:])
2020-02-20T21:14:57.7569275Z   File "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 547, in main
2020-02-20T21:14:57.7569540Z     return gyp_main(args)
2020-02-20T21:14:57.7569879Z   File "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 532, in gyp_main
2020-02-20T21:14:57.7570124Z     generator.GenerateOutput(flat_list, targets, data, params)
2020-02-20T21:14:57.7570482Z   File "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py", line 2034, in GenerateOutput
2020-02-20T21:14:57.7570778Z     sln_projects, project_objects, flat=msvs_version.FlatSolution())
2020-02-20T21:14:57.7571152Z   File "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py", line 1791, in _GatherSolutionFolders
2020-02-20T21:14:57.7571430Z     return _DictsToFolders('', root, flat)
2020-02-20T21:14:57.7571800Z   File "C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\gyp\pylib\gyp\generator\msvs.py", line 1744, in _DictsToFolders
2020-02-20T21:14:57.7572098Z     for folder, contents in bucket.items():
2020-02-20T21:14:57.7572436Z AttributeError: 'MSVSProject' object has no attribute 'items'
2020-02-20T21:14:57.7693642Z gyp ERR! configure error 
2020-02-20T21:14:57.7701952Z gyp ERR! stack Error: `gyp` failed with exit code: 1
2020-02-20T21:14:57.7709701Z gyp ERR! stack     at ChildProcess.onCpExit (C:\hostedtoolcache\windows\node\12.16.1\x64\node_modules\npm\node_modules\node-gyp\lib\configure.js:351:16)
2020-02-20T21:14:57.7721504Z gyp ERR! stack     at ChildProcess.emit (events.js:311:20)
2020-02-20T21:14:57.7747689Z gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
2020-02-20T21:14:57.7748488Z gyp ERR! System Windows_NT 10.0.17763
2020-02-20T21:14:57.7750189Z gyp ERR! command "C:\\hostedtoolcache\\windows\\node\\12.16.1\\x64\\node.exe" "C:\\hostedtoolcache\\windows\\node\\12.16.1\\x64\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
2020-02-20T21:14:57.7750876Z gyp ERR! cwd d:\a\testing-workshop-cypress\testing-workshop-cypress\node_modules\fsevents
2020-02-20T21:14:57.7751508Z gyp ERR! node -v v12.16.1
2020-02-20T21:14:57.7751774Z gyp ERR! node-gyp -v v5.0.5
2020-02-20T21:14:57.7752014Z gyp ERR! not ok 
2020-02-20T21:14:58.1691740Z 
2020-02-20T21:14:58.1692622Z > testing-workshop-cypress@1.0.0 postinstall d:\a\testing-workshop-cypress\testing-workshop-cypress
2020-02-20T21:14:58.1692984Z > npm install --prefix todomvc
2020-02-20T21:14:58.1693146Z 
2020-02-20T21:15:08.8383764Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\is-ci -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\is-ci\bin.js
2020-02-20T21:15:08.8452849Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\mime -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\mime\cli.js
2020-02-20T21:15:08.8568865Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\semver -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\semver\bin\semver.js
2020-02-20T21:15:08.8647072Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\rc -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\rc\cli.js
2020-02-20T21:15:08.8711594Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\sshpk-verify -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\sshpk\bin\sshpk-verify
2020-02-20T21:15:08.8712447Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\sshpk-sign -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\sshpk\bin\sshpk-sign
2020-02-20T21:15:08.8713034Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\sshpk-conv -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\sshpk\bin\sshpk-conv
2020-02-20T21:15:08.8753815Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\uuid -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\uuid\bin\uuid
2020-02-20T21:15:08.8791519Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\which -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\which\bin\which
2020-02-20T21:15:08.8872092Z d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\json-server -> d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\json-server\lib\cli\bin.js
2020-02-20T21:15:08.9793690Z 
2020-02-20T21:15:08.9794776Z > testing-workshop-cypress@1.0.0 postinstall d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\testing-workshop-cypress
2020-02-20T21:15:08.9795166Z > npm install --prefix todomvc
2020-02-20T21:15:08.9795350Z 
2020-02-20T21:15:18.4016594Z npm WARN cypress-plugin-snapshots@1.2.9 requires a peer of cypress@^3.4.1 but none is installed. You must install peer dependencies yourself.
2020-02-20T21:15:18.4019952Z 
2020-02-20T21:15:18.4883726Z npm ERR! code ENOENT
2020-02-20T21:15:18.4884352Z npm ERR! syscall rename
2020-02-20T21:15:18.4884847Z npm ERR! path d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\testing-workshop-cypress\todomvc\node_modules\@sindresorhus\is
2020-02-20T21:15:18.4885677Z npm ERR! dest d:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\testing-workshop-cypress\todomvc\node_modules\@sindresorhus\.is.DELETE
2020-02-20T21:15:18.4886002Z npm ERR! errno -4058
2020-02-20T21:15:18.4886862Z npm ERR! enoent ENOENT: no such file or directory, rename 'd:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\testing-workshop-cypress\todomvc\node_modules\@sindresorhus\is' -> 'd:\a\testing-workshop-cypress\testing-workshop-cypress\todomvc\node_modules\testing-workshop-cypress\todomvc\node_modules\@sindresorhus\.is.DELETE'
2020-02-20T21:15:18.4887269Z npm ERR! enoent This is related to npm not being able to find a file.
2020-02-20T21:15:18.4887784Z npm ERR! enoent 
2020-02-20T21:15:18.5395739Z 
2020-02-20T21:15:18.5396318Z npm ERR! A complete log of this run can be found in:
2020-02-20T21:15:18.5396945Z npm ERR!     C:\Users\runneradmin\.npm\_logs\2020-02-20T21_15_18_491Z-debug.log
2020-02-20T21:15:19.1936835Z npm WARN cypress-plugin-snapshots@1.2.9 requires a peer of cypress@^3.4.1 but none is installed. You must install peer dependencies yourself.
2020-02-20T21:15:19.1939703Z 
2020-02-20T21:15:19.2058376Z npm ERR! code ELIFECYCLE
2020-02-20T21:15:19.2061828Z npm ERR! errno 4294963238
2020-02-20T21:15:19.2062302Z npm ERR! testing-workshop-cypress@1.0.0 postinstall: `npm install --prefix todomvc`
2020-02-20T21:15:19.2062835Z npm ERR! Exit status 4294963238
2020-02-20T21:15:19.2063315Z npm ERR! 
2020-02-20T21:15:19.2063608Z npm ERR! Failed at the testing-workshop-cypress@1.0.0 postinstall script.
2020-02-20T21:15:19.2064001Z npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-02-20T21:15:19.2373401Z 
2020-02-20T21:15:19.2377047Z npm ERR! A complete log of this run can be found in:
2020-02-20T21:15:19.2384157Z npm ERR!     C:\Users\runneradmin\.npm\_logs\2020-02-20T21_15_19_212Z-debug.log
2020-02-20T21:15:19.2694866Z npm ERR! code ELIFECYCLE
2020-02-20T21:15:19.2695517Z npm ERR! errno 4294963238
2020-02-20T21:15:19.2706464Z npm ERR! testing-workshop-cypress@1.0.0 postinstall: `npm install --prefix todomvc`
2020-02-20T21:15:19.2706997Z npm ERR! Exit status 4294963238
2020-02-20T21:15:19.2707225Z npm ERR! 
2020-02-20T21:15:19.2707585Z npm ERR! Failed at the testing-workshop-cypress@1.0.0 postinstall script.
2020-02-20T21:15:19.2708013Z npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-02-20T21:15:19.3308090Z 
2020-02-20T21:15:19.3311441Z npm ERR! A complete log of this run can be found in:
2020-02-20T21:15:19.3312639Z npm ERR!     C:\Users\runneradmin\.npm\_logs\2020-02-20T21_15_19_291Z-debug.log
2020-02-20T21:15:19.3582105Z Error: The process 'C:\hostedtoolcache\windows\node\12.16.1\x64\npm.cmd' failed with exit code 4294963238
2020-02-20T21:15:19.3582645Z     at ExecState._setResult (d:\a\_actions\cypress-io\github-action\v1\dist\index.js:1188:25)
2020-02-20T21:15:19.3582999Z     at ExecState.CheckComplete (d:\a\_actions\cypress-io\github-action\v1\dist\index.js:1171:18)
2020-02-20T21:15:19.3583331Z     at ChildProcess.<anonymous> (d:\a\_actions\cypress-io\github-action\v1\dist\index.js:1071:27)
2020-02-20T21:15:19.3583599Z     at ChildProcess.emit (events.js:210:5)
2020-02-20T21:15:19.3586109Z     at maybeClose (internal/child_process.js:1021:16)
2020-02-20T21:15:19.3586682Z     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
2020-02-20T21:15:19.3589588Z ##[error]The process 'C:\hostedtoolcache\windows\node\12.16.1\x64\npm.cmd' failed with exit code 4294963238
```

Tried AppVeyor and GH Action

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -6,6 +6,9 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
       # Install NPM dependencies, cache them correctly
       # and run all Cypress tests
       - name: Cypress run

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -1,0 +1,16 @@
+name: windows
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      # Install NPM dependencies, cache them correctly
+      # and run all Cypress tests
+      - name: Cypress run
+        uses: cypress-io/github-action@v1
+        with:
+          start: npm start
+          wait-on: 'http://localhost:3000'
+          spec: 'cypress/integration/07-ci/*'

